src/validation.cpp

```diff
@@ -2381,10 +2484,10 @@ bool QtumTxConverter::parseEthTXParams(EthTransactionParams& params){
             stack.pop_back();
             receiveAddress = dev::Address(vecAddr);
         }
-        if(stack.size() < 4)
+        if(stack.size() < correctedStackSize(4))
             return false;
 
-        if(stack.back().size() < 1){
+        if(stack.back().size() < correctedStackSize(1)){
```

Comment:
Likely typo, this prevents OP_SENDER scriptPubKey from having a code size < 4 which prevents the typical pattern for calling the fallback function of contract. For example, the following fails with qip-5 activated (the error is in the debug log):
```qtum-cli -regtest createcontract "00" 100000 0.0000004 $(qtum-cli -regtest getnewaddress)```

Reply:
Yes typo, fix added.